starter-code/server.js

```diff
@@ -146,7 +151,7 @@ app.listen(PORT, function() {
 ////////////////////////////////////////
 function loadArticles() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // number 3 from diagram. loadAll() method on line 35. it's reading the data for the table
```

Comment:
this function is not interacting with `loadAll()`

Reply:
it's also checking if there are 0 records in the table and then doing the create CRUD method